pyproject.toml

```diff
@@ -13,9 +13,9 @@ keywords = [
   "sc2",
   "starcraft 2",
 ]
-license = {text = "MIT"}
-authors = [{name = "Kevin Leung", email = "kkleung89@gmail.com"}]
-requires-python = ">=3.7"
```

Comment:
Are there specific changes that dropped support for python 3.7 and python 3.8, or is this just tidying up because those versions are end-of-life?

Reply:
Removing end of life Python encourages users to avoid relying on unsafe software.